libraries/AP_Common/missing/termios.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+#include_next <termios.h>
+
+#if defined(__ANDROID__) && !defined(HAVE_TERMIOS_CFSETSPEED)
+
```

Comment:
The check for `__ANDROID__` should not be needed...  `HAVE_TERMIOS_CFSETSPEED` will already tell if we need to implement.this missing function.

Reply:
yes, this was added to compile without using waf, for that reason i put the double check. On build system like embarcadero is more difficult to build with waf internally, on Urus Studio this is not a problem.

Please doesn't merge it meanwhile, i have a better solution for this one, i will push it until URUS SHAL already working at 100% on Firemonkey.

Reply:
In the other build system you could simply let it undefined (hardcoded).  Here we only support waf

Reply:
yes, i know that, __ANDROID__ was removed indeed, i will push the fix.